syntax = "proto3";
import "google/protobuf/empty.proto";


message AddTrackToPlaylistRequest {
  string playlist_id = 1;
  string track_id = 2;
  string user_id = 3;
}

service PlaylistCommandService {
  rpc AddTrackToPlaylist (AddTrackToPlaylistRequest) returns (google.protobuf.Empty);
}

